src/mod_roster.erl

```diff
@@ -886,7 +886,7 @@ send_presence_type(From, To, Type) ->
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
-%% used for testing only - should be removed
+%% used by ejabberdctl
```

Comment:
As far as I noticed, the script is now named `mongooseimctl`, unless you mean the suite `ejabberdctl_SUITE`?

Reply:
True. It is somewhat confusing because all test suites still call it `ejabberdctl`.